scripts/eq_translations_check.py

```diff
@@ -10,8 +10,10 @@
 version = f"v{eq_translations.__version__}"
 
 try:
-    response = requests.get("https://api.github.com/repos/ONSdigital/eq-translations/releases").json()
-    latest_tag = response[0]['tag_name']
+    response = requests.get(
+        "https://api.github.com/repos/ONSdigital/eq-translations/releases"
+    ).json()
+    latest_tag = response[0]["tag_name"]
 
     if version == latest_tag:
         logger.error("eq-translations up to date")
```

Comment:
This should be `.error`. Do we even want to log when its all good?. Maybe only log when it needs updating?

Reply:
Removed, thanks!👍